revolv/project/models.py

```diff
@@ -291,17 +325,18 @@ def has_owner(self, ambassador):
 
     def approve_project(self):
         self.project_status = Project.ACTIVE
-        self.start_date = datetime.date.today()
+        if self.start_date is None:
```

Comment:
Don't think so, since end_date can technically be edited by the project edit page. Basically we need this because you can EITHER select a start date via the edit page OR leave it blank and have it auto-fill to be the date you press "approve"